### ISSUE TITLE
truecolor status

### ISSUE THREAD

**Issue by 4c3e** [termbox/termbox2#13]
Hello! I am trying to use this library to make a truecolor unicode art editor, and have been running into trouble trying to set fg and bg colors based on a hex value. I noticed there was no section in the comment above `tb_set_output_mode` for truecolor also, is this implementation still in process? Would love to help if that is the case. 

Here is how I am using tb_set_cell(): 

```c
  tb_set_input_mode(5);
  uint32_t bg = 0x88B8A0;
  uint32_t fg = 0xa281c6;

  // draw 'h' at some coord x,y
  tb_set_cell(x, y, 'h', fg, bg);

  tb_present();
```

**Comment by adsr** [comment]
Hello, I think your example code will work if you replace `tb_set_input_mode(5)` with `tb_set_output_mode(TB_OUTPUT_TRUECOLOR)`.

Thank you for pointing out the missing docs for `TB_OUTPUT_TRUECOLOR` above `tb_set_output_mode`. I'll add that.

**Comment by 4c3e** [comment]
Ah my mistake! Works great now, thank you so much for the quick response.

**Comment by adsr** [comment]
No problem. I suspect there aren't many people using true-color mode so you may find bugs. I was playing around with it tonight and noticed at least one (fixed in 775b6b4).
